frontend_tests/zjsunit/index.js

```diff
@@ -40,7 +40,17 @@ function immediate(f) {
 // Find the files we need to run.
 const files = process.argv.slice(2);
 if (files.length === 0) {
-    throw new Error("No tests found");
+    // This shouldn't happen.
+    // For example, in the case that you pass individual files to
+    // parallel_index_runner, via ./tools/test-js-with-node example.js,
+    // you'll also pass parallel = 1.
+    // The only other case where this is possible is where you've
+    // removed all or almost all test files from /node_tests/
+    console.info(
+        "one or more child processes did not receive any files, one possibility is:\n" +
+            "Number of processes > number of test files.",
```

Comment:
Or, we could not have that bug—just avoid creating empty groups in `create_file_groups`.